src/multipleQuestions.json

```diff
@@ -0,0 +1 @@
+[] 
```

Comment:
Great job adding this new file. Looks like we need to add some questions into this JSON. Luckily, we already have a template for what a question looks like. Copy the question data object from question.js, and use that structure to make three questions in this JSON.